gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -283,6 +291,40 @@ def get_plumbing_info(self, context):
                   'service_type': service_type})
         return plumbing_request
 
+    def _check_for_fw_vpn_sharing(self, context, service_type):
+        shared_svc_type = pconst.FIREWALL
+        if service_type == pconst.FIREWALL:
+            shared_svc_type = pconst.VPN
+        return self._is_service_type_in_chain(context, shared_svc_type)
+
+    def _get_service_type(self, profile):
+        if profile['service_type'].endswith('_HA'):
+            service_type = profile['service_type'][:-3]
```

Comment:
HA is not yet there right. We don't need this if block. We can have this function for place holder and return profile['service_type']